editor/components/block-list/block.js

```diff
@@ -350,11 +350,13 @@ class BlockListBlock extends Component {
 		const { isHovered, isSelected, isMultiSelected, isFirstMultiSelected, focus } = this.props;
 		const showUI = isSelected && ( ! this.props.isTyping || ( focus && focus.collapsed === false ) );
 		const { error } = this.state;
+		const isReusableBlock = blockType.name === 'core/block';
```

Comment:
Elsewhere we've limited our awareness to specific block types via abstraction, e.g. `getDefaultBlockName`, `getUnknownTypeHandlerName`. Should we need to consider that here? Maybe not the case we'd ever expect it to change.

Reply:
Couldn't hurt! I moved this logic out to `isReusableBlock` in 95a28a37adea3afeb44fb2b9c068c9e1fc61fb31.